.github/actions/build-vsix/action.yml

```diff
@@ -1,11 +1,6 @@
 name: 'Build VSIX'
 description: "Build the extension's VSIX"
 
-inputs:
-  package_command:
-     description: 'Package.json package script'
-     required: false
-     default: 'package'
```

Comment:
We don't need any of these anymore as we now use the azure pipelines for releases.